### DISCUSSION TITLE
URL Attribute not working

### DISCUSSION THREAD

**Discussion by krismanning** [livewire/livewire#8081] [1. Help]
HI,

Im having an issue where the URL attribute doesnt seem to work. Despite setting it the url in the browser isnt updating/changing at all. The component itself all works fine apart from that one thing.

Using the older QueryString method still works fine.

Any idea what I'm doing wrong? (MacOS using Herd, Php8.2)

`use App\Models\Role;
use Livewire\Attributes\Url;
use Livewire\Component;
use App\Models\Attendance;
use Livewire\WithPagination;
use Illuminate\Support\Facades\Auth;

class AttendanceFilter extends Component
{

    use WithPagination;

    #[Url]
    public $search = '';
    public $roles;
    #[Url]
    public $role = null;
    #[Url]
    public $activity;
    #[Url]`

**Comment by PhiloNL** [comment]
Can you re-create an example using https://wirebox.app/ or post the code of the entire example that shows the problem (remove anything that is not related to the problem itself, and make it as simple as possible for us to test)

**Reply by ellastinnes1** [reply]
This is the same issue I'm getting in a private repository. I'll extract to wirebox

**Reply by ellastinnes1** [reply]
Just to follow up, I've reinstalled in Livewire and the #[Url] attributes are now working.

**Comment by damib** [comment]
Hi
I had the same problem recently, and I've discovered there is a possible conflict with the Alpine included in the latest livewire versions and the alpine usually configured in laravel. It is like Alpine is included twice, I've just commented the alpine section in /resources/js/app.js and the problem (and some other strange behaviors) disappears (and Alpine is still working fine), Url starts working and everything is fine.
